app/mqtt/MqttClient.py

```diff
@@ -162,7 +187,7 @@ async def on_message(self, client, topic, payload, qos, properties):
             get_id = (topic.split("/"))[2]
             device_id = (get_id.split("_"))[0]
             endpoint_id = (get_id.split("_"))[1]
-            await Light.put_level_cmd(tydom_client=self.tydom, device_id=device_id, light_id=endpoint_id,
+            await Light.put_level_cmd(tydom_client=self.tydom, device_id=device_id, cover_id=endpoint_id,
```

Comment:
Hello
I think this part is causing side effect for lights and should not be change.